internal/parser/simple_parser_rules.go

```diff
@@ -676,3 +678,38 @@ func (p *simpleParser) parseAttachDatabaseStmt(r reporter) (stmt *ast.AttachStmt
 	p.consumeToken()
 	return
 }
+
+// parseDetachDatabaseStmt parses statements of the form :
+// DETACH DATABASE schema-name
```

Comment:
Not only this, also `DETACH schema-name`

Reply:
```suggestion
// DETACH DATABASE schema-name
// DETACH schema-name
```

Reply:
Ive written the same for `ATTACH`, since theyre a `*` I thought it can be written that way.

I can be more precise and say `ATTACH (DATABASE)* schema-name1 ?

Reply:
no because that means that database may occur 0 or more times, so `ATTACH DATABASE DATABASE DATABASE schema` would also be valid. You would have to use a `?`